config.go

```diff
@@ -0,0 +1,33 @@
+package gogpt
+
+import (
+	"net/http"
+)
+
+const (
+	apiURLv1                       = "https://api.openai.com/v1"
+	defaultEmptyMessagesLimit uint = 300
+)
+
+// ClientConfig is a configuration of a client.
+type ClientConfig struct {
+	authToken string
```

Comment:
Why is `authToken` private, but the other fields are public?

Reply:
Great question! I made it public at first but then figured that I couldn't imagine a case where it needs to be modified or explicitly accessed. 

I also thought it would prevent accidental logging, but it seems that it is not the case.

Should I make it public?
🤔 🤔 🤔

Reply:
@mwillfox btw, it leaks if you do `%v` printing of the struct